dotcom-rendering/src/web/components/TableOfContents.importable.tsx

```diff
@@ -95,24 +86,21 @@ export const TableOfContents = ({ tableOfContents, format }: Props) => {
 	const palette = decidePalette(format);
 	const [open, setOpen] = useState(tableOfContents.length < 5);
 
-	// The value for data-link-name is evaluated at the time when the component renders,
-	// So at the time when user clicks the table (onToggle is triggered),
-	// the old value of open(before the click event) is used. As a result we need to
-	// use toc-close for when open is true and toc-expand for when it's false
 	return (
 		<details
 			open={open}
 			css={detailsStyles}
 			data-component="table-of-contents"
 		>
 			<summary
-				onClick={() => {
+				onClick={(e): void => {
+					e.preventDefault();
 					setOpen(!open);
 				}}
 				data-link-name={
 					open
-						? 'table-of-contents-close'
-						: 'table-of-contents-expand'
+						? 'table-of-contents-expand'
```

Comment:
Not a blocker - but any reason not to use `open` as it's used everywhere else including the native element?

Reply:
Good idea, I chatted with Guy from the data insight team and he was happy to use `table-of-contents-open` instead of `table-of-contents-expand` 👍